CHANGELOG.md

```diff
@@ -1,3 +1,4 @@
+bluetooth: fix crash on MacOS when doing first download from a new BLE device
```

Comment:
@dirkhh: Do we want to retire `CHANGELOG.md`, or at least remove it from the instructions for pull requests? With our new approach of doing CICD releases it does not seem to serve any particular function any more.